### PR TITLE
Fixes #25159 - Add Ansible variables to hosts form

### PR DESCRIPTION
**Pull request by xprazak2**
Needs:
https://github.com/theforeman/foreman_ansible/pull/313
~https://github.com/theforeman/foreman_ansible/pull/374~
https://github.com/theforeman/foreman/pull/8194
~https://github.com/theforeman/foreman/pull/8231~

I'll rebase once the required PRs are merged.

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Right, can you rebase please so we get to a clean starting point? :)

File: webpack/__mocks__/foremanReact/common/I18n.js [hunk 1]
```diff
@@ -1 +1,3 @@
+export { sprintf } from 'jed';
```
Comment (ezr-ondrej) [inline webpack/__mocks__/foremanReact/common/I18n.js:1]: Why do we need the real version in a mock?

Comment (xprazak2) [reply webpack/__mocks__/foremanReact/common/I18n.js:1]: Correct formatting of translated messages: 

```js
sprintf(__('Invalid value, expected one of: %s'), lookupKey.validatorRule)
```

Comment (ezr-ondrej) [reply webpack/__mocks__/foremanReact/common/I18n.js:1]: Can't we just ignore additional arguments to the sprintf?
We are not really testing the strings, are we? We can just have your example to result in `'Invalid value, expected one of: %s'`, right? we'd just get this in the snapshots, but that should be fine.... Or are we depending on it with some tests?

**Comment by xprazak2** [comment]
Closing in favor of https://github.com/theforeman/foreman_ansible/pull/445, https://github.com/theforeman/foreman_ansible/pull/444 and https://github.com/theforeman/foreman_ansible/pull/458

**Pull request closed without merging**